lib/plugins/inspec-habitat/Rakefile

```diff
@@ -0,0 +1,34 @@
+require 'rake/testtask'
+
+desc 'Run lint, unit, and functional tests'
+task test: ['test:lint', 'test:unit', 'test:functional']
+
+namespace :test do
+  Rake::TestTask.new(:lint) do |t|
+    t.description = 'Lint all Ruby files'
+    begin
+      require 'rubocop/rake_task'
+      RuboCop::RakeTask.new(:lint)
+    rescue LoadError
+      puts 'Please install the `rubocop` gem for lint tests.'
+    end
+  end
+
+  Rake::TestTask.new(:unit) do |t|
+    t.description = 'Run all unit tests'
+    t.test_files = FileList['test/unit/**/*_test.rb']
+    t.warning = false
```

Comment:
FWIW, I've been disabling `t.warning` - it is rarely useful and always very noisy, due to method redefs in dependent libraries.

Reply:
Went ahead and remove the file, but noted!